flake.nix

```diff
@@ -14,8 +14,63 @@
         overlays = [ (import rust-overlay) ];
         pkgs = import nixpkgs { inherit system overlays; };
         shell = import ./shell.nix { inherit pkgs; };
+        inherit (pkgs) lib rustPlatform;
       in
       {
+        packages = {
+          prisma-engines = rustPlatform.buildRustPackage
+            {
+              name = "prisma-engines";
+              src = builtins.path {
+                path = ./.;
+                name = "prisma-engines-workspace-root-path";
+              };
+              cargoLock = {
+                lockFile = ./Cargo.lock;
+                # Hashes are required for git dependencies.
+                outputHashes = {
+                  "barrel-0.6.6-alpha.0" = "sha256-USh0lQ1z+3Spgc69bRFySUzhuY79qprLlEExTmYWFN8=";
```

Comment:
Could we have a `updateHashes` command to automatically change these?

Reply:
Yep that's possible, maybe there's already something in nixpkgs for it.